src/Toolkit.Forms/SearchView/SearchView.Appearance.cs

```diff
@@ -0,0 +1,201 @@
+// /*******************************************************************************
+//  * Copyright 2012-2018 Esri
+//  *
+//  *  Licensed under the Apache License, Version 2.0 (the "License");
+//  *  you may not use this file except in compliance with the License.
+//  *  You may obtain a copy of the License at
+//  *
+//  *  http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  *   Unless required by applicable law or agreed to in writing, software
+//  *   distributed under the License is distributed on an "AS IS" BASIS,
+//  *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  *   See the License for the specific language governing permissions and
+//  *   limitations under the License.
+//  ******************************************************************************/
+using System.ComponentModel;
+using Esri.ArcGISRuntime.Toolkit.UI.Controls;
+using Xamarin.Forms;
+using Grid = Xamarin.Forms.Grid;
+using XForms = Xamarin.Forms.Xaml;
+
+namespace Esri.ArcGISRuntime.Toolkit.Xamarin.Forms
+{
+    public partial class SearchView : TemplatedView, INotifyPropertyChanged
+    {
+#pragma warning disable SA1306, SA1310, SX1309 // Field names should begin with lower-case letter
+        private Entry? PART_Entry;
```

Comment:
There were named on the assumption that there would be a `TemplatePartAttribute` equivalent for Xamarin.Forms, but there doesn't appear to be. Am I missing something?